docs/design_flaws.md

```diff
@@ -770,3 +771,11 @@ Edit `GetForestTreeFrame`:
 +	add a
  	ret
 ```
+
+
+## The 6-bit caught level can only record up to level 63
+
+The 6-bit field that is normally used to record the caught level of Pokémon can only record up to level 63 normally.
+
+**Possible Solution:** Repurpose a free bit from `MON_LEVEL` (1 free bit), `MON_EXP` (3 free bits), etc.
+An example of such a fix was used in https://github.com/thegsproj/pokegscrystal/pull/8.
```

Comment:
```suggestion
An example of such a fix was used in https://github.com/thegsproj/pokegscrystal/pull/8.
```

Reply:
Individual examples can go in the wiki's Branches page, or a dedicated page to "Improving the caught data", or whatever.

Reply:
I agree with Rangi that example fixes should be in Wiki Pages.